cabal-testsuite/PackageTests/NewBuild/CmdRun/Terminate/cabal.test.hs

```diff
@@ -19,7 +19,7 @@ without forking in the future.)
 main :: IO ()
 main = do
   skipIfWindows "depends on `unix`"
-  cabalTest $ do
+  flakyIfCI 11087 $ cabalTest $ do
```

Comment:
Actually, those two need to be reversed: the other usages I see are `cabalTest $ flakyIfCI <pr> $ do …`.